WinUIGallery/Samples/ControlPages/ContentDialogPage.xaml.cs

```diff
@@ -28,21 +35,50 @@ private async void ShowDialog_Click(object sender, RoutedEventArgs e)
         dialog.CloseButtonText = "Cancel";
         dialog.DefaultButton = ContentDialogButton.Primary;
         dialog.Content = new ContentDialogContent();
-        dialog.RequestedTheme = (VisualTreeHelper.GetParent(sender as Button) as StackPanel).ActualTheme;
```

Comment:
Why was this removed? I think we need to keep this and also use this in the other function.